src/Grid.js

```diff
@@ -1,5 +1,30 @@
+import Tile from './Tile.js'
 export default class Grid {
-  constructor() {
+  constructor(size) {
+  	this.size = size;
+  	this.grid = [];
+  	while(this.grid.push([]) < size);
+   	for (var i = 0; i < this.size; i++) {

```

Comment:
change all instances of `var` to `let` there's a lot of subtle javascript bugs that you can avoid by using `let` instead of `var`.